database/index.spec.js

```diff
@@ -0,0 +1,35 @@
+const mongoose = require('mongoose');
+const connection = require('./index.js');
+const request = require("supertest");
+const app = require("../server/server.js");
+const Model = connection.Booking;
+const databaseName = 'booking';
+
+describe('Test the seedDb method', () => {
+  beforeAll(() => {
+    const url = `mongodb://localhost:3002/${databaseName}`
+    mongoose.connect(url, { useNewUrlParser: true })
+  });
+
+  afterAll((done) => {
+    mongoose.connection.close(() => {
+      done();
+    });
+  });
+
+});
+
+test('should store booking data to database', async done => {
+  const booking = await Model.findOne({ campId: 0 })
+  await request(app)
+  .get('/booking?campId=0')
+  .then((data) => {
+    let site = JSON.parse(data.text);
+    expect(Array.isArray(site.booked)).toBeTruthy();
```

Comment:
does truthy mean that the array exists? that is to say would there be a case where this test would pass because the array exists but it doesnt contain the information that you would expect. The answer may be no, but I am sometimes leary of using truthy on arrays... I tend to use something like `array.length !== 0` but this is only to check that there are items in the array. just a thought.

Reply:
I need to make this one way more specific. As I'm learning jest, I'm steering for lowest hanging fruit but this should really be an array of nested arrays containing date ranges that are unavailable for each site. Need to go back and improve this  one. Thank you!